Source/RenderStream/Private/SceneSelector_StreamingLevels.cpp

```diff
@@ -117,7 +121,7 @@ bool SceneSelector_StreamingLevels::ValidateLevel(uint32_t sceneId)
     RenderStreamLink::RemoteParameters& parameters = Schema().scenes.scenes[sceneId];
     const SchemaSpec& spec = m_specs[sceneId];
     UE_LOG(LogRenderStream, Log, TEXT("SceneSelectorStreamingLevels: Validating schema for %s with %d parameters"), UTF8_TO_TCHAR(parameters.name), parameters.nParameters);
-    AActor* levelRoot = spec.streamingLevel ? spec.streamingLevel->GetLevelScriptActor() : nullptr;
+    AActor* levelRoot = spec.streamingLevel ? reinterpret_cast<AActor*>(spec.streamingLevel->GetLevelScriptActor()) : nullptr;
```

Comment:
Why do we not use a static_cast here?

Reply:
Initially it was complaining that it couldn't do a static_cast for those types but after I included the header file for the base class in the file it started working.